service/src/main/java/io/camunda/service/AuthorizationServices.java

```diff
@@ -82,8 +90,14 @@ public List<String> getAuthorizedResources(
   }
 
   public List<String> getAuthorizedApplications(final Set<String> ownerIds) {
-    return getAuthorizedResources(
-        ownerIds, PermissionType.ACCESS, AuthorizationResourceType.APPLICATION);
+    if (securityConfiguration.getAuthorizations().isEnabled()) {
```

Comment:
💭 its preference of course but I would prefer this check to be the other way around so we have a return early style approach “if not enabled return *”, then the rest of the method is just normal/typical operation.